Cargo.toml

```diff
@@ -119,12 +119,13 @@ etcd-client = { version = "0.13" }
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "7ca323090b3ae8faf2c15036b7f41b7c5225cf5f" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "e52fa3e63c77545618328781e11178eb2cfc45c5" }
 humantime = "2.1"
 humantime-serde = "1.1"
 itertools = "0.10"
 lazy_static = "1.4"
-meter-core = { git = "https://github.com/GreptimeTeam/greptime-meter.git", rev = "80b72716dcde47ec4161478416a5c6c21343364d" }
+meter-core = { git = "https://github.com/GreptimeTeam/greptime-meter.git", rev = "488844b4702916cc6f7d04898fb0b837afce4ca7" }
+meter-macros = { git = "https://github.com/GreptimeTeam/greptime-meter.git", rev = "488844b4702916cc6f7d04898fb0b837afce4ca7" }
```

Comment:
`meter-macros` is placed deliberately in a different section  because we want to use different features in other version. I'm not sure place the dependency here is a good option.

Reply:
I see. Let me revert this.